console/src/main/webapp/manager/app/assets/lang/en.json

```diff
@@ -284,5 +284,7 @@
   "date.3month"           : "Last 3 months",
   "date.year"             : "Last year",
   "date.custom"           : "Custom range",
-  "date.to"               : "to"
+  "date.to"               : "to",
+  "sasl.externallogin"    : "External login",
```

Comment:
Where is `sasl.externallogin` being used ?

Reply:
anywhere